### DISCUSSION TITLE
The input file Job_model30.fds does not exist

### DISCUSSION THREAD

**Discussion by Y-Bruce-G** [firemodels/fds#13583] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/97e94ad5-d4d5-4e59-8e12-ce93dc67634a)
Why does this problem occur
![image](https://github.com/user-attachments/assets/b2bec952-ae0d-4c4e-8514-cbbaa63c05af)
Problems arise with both parallel and single-core operations

**Comment by TristanHehnen** [comment, marked as answer]
In you first screenshot it looks like the file on disk is called `model30.fds`, but you try to call `Job_model30.fds`. It tells you that the latter does not exist.

In the second screenshot the error seems to be related to MPI and hydra, not the existence of any given FDS input file. Maybe your process does not have write permission to the directory you want to write to?

**Comment by Y-Bruce-G** [comment]
Thank you for your answer.
If I don't add job_ to the first question, there will be a second
screenshot problem. As you said, it may indeed be a permission problem.
According to the general situation, how should I give fds permission? My
other computers have not encountered this problem.


FireTristan ***@***.***>于2024年10月16日 周三16:26写道：
